packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -191,16 +192,9 @@ function nodeDataToMapTree(
 }
 
 /**
- * Throws a UsageError if mapTree is out of schema.
- * @remarks
- * This requires mapTree to have all required default values,
- * like identifiers for identifier fields.
+ * Throws a UsageError if maybeError indicates a tree is out of schema.
  */
-export function inSchemaOrThrow(
-	schemaValidationPolicy: SchemaAndPolicy,
-	mapTree: MapTree,
-): void {
-	const maybeError = isNodeInSchema(mapTree, schemaValidationPolicy);
+export function inSchemaOrThrow(maybeError: SchemaValidationErrors): void {
```

Comment:
This utility was created to help with future node creation APis. They ended up wanting to use isFieldInSchema, so this refactoring makes it possible to use this helper with both.